example/scripts/generate_ingest/generate_files.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable node/no-extraneous-require */
+/* eslint-disable no-console */
+
+const { s3PutObject } = require('@cumulus/aws-client/S3');
+const { randomStringFromRegex } = require('@cumulus/common/test-utils');
+
+const granuleRegExp = 'MOD09GQ\\.A[\\d]{7}\\.[\\w]{6}\\.006\\.[\\d]{13}';
```

Comment:
Why not require the RX from the collection file?

Reply:
Good suggestion: 8ab225707